tests/functional/spec/components/radio/radio_open_answers.spec.js

```diff
@@ -0,0 +1,28 @@
+const RadioOpenTruePage = require('../../../generated_pages/radio_open/radio-open-true.page.js');
+const RadioOpenFalsePage = require('../../../generated_pages/radio_open/radio-open-false.page.js');
+const RadioOpenNonePage = require('../../../generated_pages/radio_open/radio-open-none.page.js');
+
+describe('Given I start a Radio survey with a write-in option', function() {
+  beforeEach(function() {
+    browser.openQuestionnaire('test_radio_open.json');
+    $(RadioOpenTruePage.coffee()).click();
```

Comment:
I don't think this step should be in the `beforeEach` as it could have an effect on the behaviour of the radio options.

Reply:
I removed "beforeEach", thanks.